src/sentry/middleware/devtoolbar.py

```diff
@@ -0,0 +1,64 @@
+import logging
+
+from sentry import analytics
+from sentry.utils.http import (
+    get_api_path_from_request,
+    origin_from_request,
+    parse_id_or_slug_param,
+    query_string,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class AnalyticsMiddleware:
```

Comment:
can we call this `DevToolBarAnalytics`?

Reply:
sure, can we keep the `Middleware` suffix?

Reply:
yeah